packages/blockfrost/test/blockfrostProvider.test.ts

```diff
@@ -65,19 +79,22 @@ describe('blockfrostProvider', () => {
       }
     });
 
-    expect(response[1]).toHaveLength(2);
-    expect(response[1][0]).toMatchObject<Cardano.TxIn>({
+    expect(response.utxo[1]).toHaveLength(2);
+    expect(response.utxo[1][0]).toMatchObject<Cardano.TxIn>({
       txId: '6f04f2cd96b609b8d5675f89fe53159bab859fb1d62bb56c6001ccf58d9ac128',
       index: 0
     });
-    expect(response[1][1]).toMatchObject<Cardano.TxOut>({
+    expect(response.utxo[1][1]).toMatchObject<Cardano.TxOut>({
       address:
         'addr_test1qz2fxv2umyhttkxyxp8x0dlpdt3k6cwng5pxj3jhsydzer3jcu5d8ps7zex2k2xt3uqxgjqnnj83ws8lhrn648jjxtwq2ytjqp',
       value: {
         coins: 1_097_647,
         assets: {}
       }
     });
+
+    expect(response.delegationAndRewards.delegate).toEqual(accountsMockResponse.pool_id);
+    expect(response.delegationAndRewards.rewards).toEqual(Number(accountsMockResponse.rewards_sum));
```

Comment:
```suggestion
    expect(response.delegationAndRewards.rewards).toEqual(Number(accountsMockResponse.withdrawable_amount));
```

We're after the balance of the account, rather than total rewards received